ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -128,10 +129,41 @@ def write_record(self, record: Entity) -> None:
             self.write_table_tests(record)
         elif isinstance(record, OMetaPipelineStatus):
             self.write_pipeline_status(record)
+        elif isinstance(record, DataModelLink):
+            self.write_datamodel(record)
         else:
             logging.info(
                 f"Ignoring the record due to unknown Record type {type(record)}"
             )
+            self.write_create_request(record)
+
+    def write_create_request(self, entity_request) -> None:
```

Comment:
This should be one of the few methods run by the sink. The rest need to be cleaned up after fixing the source logic